lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -3036,83 +3019,6 @@ public function size()
     }
 
     /**
-     * Gets the EntityPersister for an Entity.
-     *
-     * @param string $entityName The name of the Entity.
-     *
-     * @return \Doctrine\ORM\Persisters\Entity\EntityPersister
-     */
-    public function getEntityPersister($entityName)
```

Comment:
I can't see how someone could use Persisters directly.
Only reasonable way to consume this API is through OnFlush event, and calling Persister methods from there is so risky I wouldn't it anyway. This method was public only because it was used everywhere (like EntityRepository)